src/framework/link/Link.jsx

```diff
@@ -28,6 +34,7 @@ Link.propTypes = {
   href: PropTypes.string,
   onClick: PropTypes.func,
   data: PropTypes.any,
+  isTargetBlank: PropTypes.bool,
```

Comment:
I hear you, I think we should go with my suggestion. At some point we might want to target specific tabs (http://stackoverflow.com/questions/6903815/how-to-open-link-url-in-specific-browser-tab-window-page).